cmd/cosign/cli/fulcio/fulcioverifier/fulcioverifier.go

```diff
@@ -37,6 +40,10 @@ import (
 // This is the CT log public key target name
 var ctPublicKeyStr = `ctfe.pub`
 
+// Setting this env variable will over ride what is used to validate
+// the SCT coming back from Fulcio.
+const altCTLogPublicKeyLocation = "CT_LOG_PUBLIC_KEY_FILE"
```

Comment:
What do you think about prefixing this with `SIGSTORE_`? We have some env vars prefixed with `COSIGN_` for cosign-specific things, and then`SIGSTORE_ROOT_FILE` and `SIGSTORE_NO_CACHE` for the TUF root itself.